assets/JS/validation.js

```diff
@@ -0,0 +1,56 @@
+export function isEmail(value, inputElement) {
+  const emailValid = value.includes("@");
+  const errMsg = inputElement.nextElementSibling;
+  if (!emailValid) {
+    errMsg.classList.add("show-msg");
+    inputElement.classList.add("error");
+  } else {
+    errMsg.classList.remove("show-msg");
+    inputElement.classList.remove("error");
+  }
+  return emailValid;
+}
+// 이메일 검사
+
+export function isNotEmpty(value, inputElement) {
+  const isInvalid = value.trim() === "";
+  const errMsg = inputElement.nextElementSibling;
+  if (isInvalid) {
+    errMsg.classList.add("show-msg");
+    inputElement.classList.add("error");
+  } else {
+    errMsg.classList.remove("show-msg");
+    inputElement.classList.remove("error");
+  }
+  return isInvalid;
+}
```

Comment:
위에 작성된 `toggleError` 가 적용되면 일괄적으로 아래처럼 변경할 수 있겠죠?

```suggestion
export function isNotEmpty(value, inputElement) {
  const isInvalid = value.trim() === "";
  toggleError(!isInvalid, inputElement);
  return !isInvalid;
}
```